spec/src/main/asciidoc/protocol-wireformat.adoc

```diff
@@ -131,8 +131,12 @@ Each provider MUST provide the REST/HTTP interaction, but MAY provide other prot
 == Health Check Procedures
 * A producer MUST support custom, application level health check procedures
 * A producer SHOULD support reasonable out-of-the-box procedures
-* A producer with no health check procedures expected or installed MUST return positive overall status (i.e. HTTP 200)
-* A producer with health check procedures expected but not yet installed MUST return negative overall status (i.e. HTTP 503)
+
+* A producer with no liveness procedures expected or installed MUST return positive overall status (i.e. HTTP 200)
+* A producer with no readiness procedures expected or installed MUST return positive overall status (i.e. HTTP 200)
```

Comment:
Regarding this statement, if there are no readiness procedures expected or installed (empty checks), wouldn't that mean that the overall Readiness status should be negative, according to the below "Empty default readiness health check responses" statement?

Reply:
So I tried to think about app servers with this. If you are just starting the server without deployments then no procedures are expected or installed and thus it should be ready after start. If you starting runtime with single app then you should expect some procedures and thus return DOWN for the duration before they are installed/processed. If users don't define any procedures (so no procedures have been installed) then you are back in this bullet and you should return UP. Isn't it clear from the bullet points?

Reply:
Oh okay, that makes sense. I think it should be more clearer from the bullet points, since it can get quite confusing with the Empty default readiness health check responses section. As a general reader/user, I would think that if the producer has no readiness procedures expected/installed, that would mean there will be an empty checks response for Readiness, which would mean the default status would be DOWN. Maybe, remove or change the word "empty" from the below section?

Reply:
ok @pgunapal, I will try to reword it.